doc/rst/usingchapel/executing.rst

```diff
@@ -181,41 +181,27 @@ variables. See :ref:`readme-main` for more
 information.
 
 
------------------------------
-Setting the Number of Locales
------------------------------
-
-For multi-locale Chapel executions, the number of locales on which to
-execute a program is specified on the executable's command-line.  This
-can be set either using the ``-nl`` flag, or by assigning to the built-in
-numLocales configuration constant using the normal mechanisms.  So, to
-execute on four locales, one could use:
-
-  .. code-block:: sh
-
-    ./myprogram -nl 4
-
-or:
-
-  .. code-block:: sh
-
-    ./myprogram --numLocales=4
-
-or:
-
-  .. code-block:: sh
-
-    ./myprogram -snumLocales=4
-
-For users running with ``CHPL_COMM=none`` (the default), only one
-locale can be used.  See :ref:`readme-multilocale` for more
-information about executing on multiple locales.
-
-Multi-locale programs often use a launcher executable to do some initial
-command-line checking before spawning the real program, which is then
-stored in a second binary named ``<original_binary_name>_real``.  See
-:ref:`readme-launcher` for more information about the launcher executable.
-
+---------------------
+Multi-locale Programs
+---------------------
+
+For multi-locale Chapel programs, the number of locales to create is specified
+on the executable's command-line.  This can be set either using the ``-nl``
+flag, or by assigning to the built-in ``numLocales`` configuration constant.
```

Comment:
This now reads to me as if I can write `numLocales = 4;` at the top of my program and then just execute as `./hello`, which I don't think is correct.

Maybe this can be clarified with an example?

```
or by assigning to the built-in ``numLocales`` configuration constant (e.g. ``-snumLocales=4`` or ``--numLocales=4``).
```

Reply:
Done, and also did some minor rewording of the entire section.